packages/eslint-plugin-pf-codemods/lib/rules/v5/cardHeader-update-api.js

```diff
@@ -8,142 +8,146 @@ module.exports = {
     const removedCardImports = imports.filter((specifier) =>
       ["CardHeaderMain", "CardActions"].includes(specifier.imported.name)
     );
-    const cardHeaderImport = imports.find((specifier) => specifier.imported.name === "CardHeader");
+    const cardHeaderImport = imports.find(
+      (specifier) => specifier.imported.name === "CardHeader"
+    );
+
+    const findParents = (node, elementNameToFind) => {
+      const parents = {
+        current: node?.parent,
+        conditionalExpression: undefined,
+      };
+
+      while (parents.current) {
+        if (parents.current.openingElement?.name?.name === elementNameToFind) {
+          return parents;
+        }
+        if (
+          ["ConditionalExpression", "LogicalExpression"].includes(
+            parents.current.type
+          )
+        ) {
+          parents.conditionalExpression = parents.current;
+        }
+
+        parents.current = parents.current.parent;
+      }
+
+      return undefined;
+    };
+    const getComponentContent = (component) => {
```

Comment:
is there no `innerHTML` or equivalent that could have achieved the same?

Reply:
Replaced it with just mapping over the node children to `getText` instead